### PR TITLE
Snap 2144 : Pulse should display the total number of cores in use by the cluster

### PR DESCRIPTION
**Pull request by snappy-sachin**
## Changes proposed in this pull request

  - Code changes for displaying Total Number of Cores configured in cluster, on Pulse UI

## Patch testing

  - Tested Manually.

## ReleaseNotes.txt changes

  - Yes.

## Other PRs 

  Spark : https://github.com/SnappyDataInc/spark/pull/89

### CODE REVIEW

File: cluster/src/main/scala/org/apache/spark/ui/SnappyDashboardPage.scala [hunk 1]
```diff
@@ -275,9 +275,22 @@ private[ui] class SnappyDashboardPage (parent: SnappyDashboardTab)
...
+    val executorToTaskSummary = parent.parent.executorsListener.executorToTaskSummary
+    val executorKeysItr = executorToTaskSummary.keySet.iterator
+    while(executorKeysItr.hasNext){
+      val execKey = executorKeysItr.next()
+      val executor = executorToTaskSummary.get(execKey)
+      totalNumExecutorCores += executor.get.totalCores
+    }
+    totalNumExecutorCores
+  }
+
```
Comment (sumwale) [inline cluster/src/main/scala/org/apache/spark/ui/SnappyDashboardPage.scala]: You could simply use SnappyContext.totalCoreCount.get()

Comment (snappy-sachin) [reply cluster/src/main/scala/org/apache/spark/ui/SnappyDashboardPage.scala]: Thanks Sumedh for suggesting that. I have made the changes now.

**Comment by sumwale** [comment]
@snappy-sachin as discussed elsewhere, need to use profile walk-through to get the total core count including for locators and leads (see the code sample sent).

**Comment by snappy-sachin** [comment]
This PR is no more relevant to new Snappy Pulse UI.
Recreating branch for it and will raise PR with Sumedhs Feedback.

**Pull request closed without merging**